README.adoc

```diff
@@ -41,8 +41,16 @@ image:https://dev.opera.com/extensions/branding-guidelines/addons_206x58_en@2x.p
  2. Open local or remote .ad, .adoc, .asc, .asciidoc files in Firefox
  3. Enjoy!
 
-WARNING: On Firefox, pages viewed directly from GitHub may not render properly. This is result of a https://bugzilla.mozilla.org/show_bug.cgi?id=1267027[Firefox bug] carrying over the restrictive content security policies (CSP) that GitHub sends to content inserted by the WebExtension.
-Please note that this issue can also occured on other websites with restictive CSP.
+[WARNING]
+====
+On Firefox, pages viewed directly from GitHub may not render properly.
+This is result of a https://bugzilla.mozilla.org/show_bug.cgi?id=1267027[Firefox bug] carrying over the Content Security Policies (CSP).
+The restrictive CSP sent by GitHub are applied to the content inserted by the WebExtension.
+
+In addition, Firefox does not trigger the extension when navigating to a `file://` from a link.
```

Comment:
On minor rewording request.

> In additiona, the extension does not automatically trigger in Firefox when navigating to a `file://` from a link.

That puts the correct subject first.

Reply:
Done :+1: